tests/operator_test.go

```diff
@@ -457,20 +507,48 @@ spec:
 		deleteAllKvAndWait(ignoreDeleteOriginalKV)
 
 		kvs := getKvList()
-
 		if len(kvs) == 0 {
 			createKv(copyOriginalKv())
 		}
-		patchOperatorVersion(tests.KubeVirtVersionTag)
+
+		modified := patchOperatorVersion(originalOperatorVersion)
+		if modified {
+			// make sure we wait until redeploymemt started
+			waitForUpdateCondition(originalKv)
+		}
+
 		waitForKv(originalKv)
-		allPodsAreReady(tests.KubeVirtVersionTag)
+		allPodsAreReady(originalKv)
 
 		if workDir != "" {
 			err = os.RemoveAll(workDir)
 			workDir = ""
 			Expect(err).ToNot(HaveOccurred())
 		}
 
+		// make sure virt deployments use shasums again after each test
+		ensureShasums()
+
+	})
+
+	Describe("should start a VM", func() {
+		It("using virt-launcher with a shasum", func() {
```

Comment:
I ran this test locally using 'make cluster-up && make cluster-sync && make functest' and no sha's are being used. the test fails for me, but it appears to work in ci? 

For some reason my environment doesn't appear to use sha by default.

Reply:
Ha, yes, actually that was something I wanted to ask:
Using shasums is currently triggered by setting the PUSH_LOG_FILE env var, see [0].
Should I set it by default in [1]?

[0] https://github.com/kubevirt/kubevirt/pull/2483/files#diff-dfb5200e5cb608be6dd572a638a6f003R196
[1] https://github.com/kubevirt/kubevirt/pull/2483/files#diff-629ce44207eb3be63928a0d226d0e672R11

Thanks for the review! Will address the other comments tomorrow.

Reply:
ah, i see now. That should have been obvious to me if I had read the description better.

I think it makes sense to set that PUSH_LOG_FILE to something in the _out dir by default.

All i care about is that both the sha path and image tag path are exercised somehow in the func-tests without any special considerations.  Having the environment setup to always use sha initially during cluster-sync is probably the easiest path forward to do that.

Reply:
> I think it makes sense to set that PUSH_LOG_FILE to something in the _out dir by default.

good idea, done

> All i care about is that both the sha path and image tag path are exercised somehow in the func-tests without any special considerations.

Yep, it is. The initial deployment is always with shasums, update tests are done with tags, and after each test we reset to the shasum based deployment (and that is tested). So we are fine here :)